playbooks/openshift-logging/private/config.yml

```diff
@@ -13,33 +13,101 @@
           status: "In Progress"
           start: "{{ lookup('pipe', 'date +%Y%m%d%H%M%SZ') }}"
 
+- name: Verify and collect ES hosts
```

Comment:
I think all of these elasticsearch/logging tasks should be in a `when: openshift_logging_install_logging | default(false) | bool` block.

Reply:
Should we then just iterate over `all` hosts for the removal case?